### ISSUE TITLE
Idea: LZAV compression algorhythm

### ISSUE THREAD

**Issue by RarogCmex** [mhx/dwarfs#198]
I have found https://github.com/avaneev/lzav
Seems easy to integrate, has MIT license. Looks like an alternative for lzhc.
So I consider to discuss it as an option for dwarfs 1.1 or so on.

**Comment by RarogCmex** [comment]
And, last but not least, this algorithm is from russian developer Aleksey Vaneev from Komi republic :ru: :smile: . I'm from Russia too.

**Comment by mhx** [comment]
I agree that it would be fairly easy to integrate. However, integrating a new algorithm also means having to support it going forward. I'd prefer to only add stuff to DwarFS that I can either fix myself (true for ricepp), or that is too big to go away (which is true for all of lzma, lz4, zstd, flac, and likely even brotli).

I'm not entirely convinced that lzav brings anything to the table that would make a massive difference in the context of DwarFS.

I'll move this to discussions for now, as I won't be integrating lzav unless I see it getting more traction.
